main.go

```diff
@@ -26,16 +27,13 @@ type Gomplate struct {
 }
 
 // RunTemplate -
-func (g *Gomplate) RunTemplate(text string, out io.Writer) {
+func (g *Gomplate) RunTemplate(text string, out io.Writer) error {
```

Comment:
This function is public API, so the signature can't change. Please remove the `error`. `panic`ing isn't the _best_ way to error out, but it's not the end of the world, and I'd _much_ rather keep API compatibility. I know of at least one user who bundles gomplate into a different tool and calls `RunTemplate` directly, so this would break them.